R/accounts.R

```diff
@@ -72,6 +76,40 @@ connectApiUser <- function(account = NULL, server = NULL, apiKey, quiet = FALSE)
   invisible()
 }
 
+
+#' @rdname connectApiUser
+#' @export
```

Comment:
You can use `@inheritParams` so you can document `snowflakeConnectionName` here, while still avoid duplicating the other arguments. See `deployDoc` for an example.